percy/providers/genericProvider.js

```diff
@@ -245,10 +245,24 @@ class GenericProvider {
   async getRegionsByElements(elementsArray, elements) {
     for (let index = 0; index < elements.length; index++) {
       try {
-        const type = await elements[index].getAttribute('class');
-        const selector = `element: ${index} ${type}`;
+        let identifier;
+        const element = elements[index];
 
-        const ignoredRegion = await this.getRegionObject(selector, elements[index]);
+        const capabilities = await this.driver.getCapabilities();
+        const PLATFORM_NAME = (capabilities.platformName || '').toLowerCase();
```

Comment:
i didn’t mean this, why were we doing a fallback here, capabilities.platformName should be there always.